src/lib/messages/push.tsx

```
@@ -1,107 +1,73 @@
-  min-height: 8px;
-  min-width: 8px;
-  fill: ${({ theme }) => theme.klerosUIComponentsWhiteBackground};
-`;
+type VariantProp = { variant: "success" | "error" | "sync" };
 
 interface PushProps extends SmallProp, VariantProp {
   title: string;
   msg?: string;
-  //eslint-disable-next-line @typescript-eslint/ban-types
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
   callback: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)